src/timesync/timesyncd-manager.c

```diff
@@ -1124,9 +1124,7 @@ int manager_new(Manager **ret) {
 
         RATELIMIT_INIT(m->ratelimit, RATELIMIT_INTERVAL_USEC, RATELIMIT_BURST);
 
-        r = manager_parse_server_string(m, SERVER_FALLBACK, NTP_SERVERS);
-        if (r < 0)
-                return r;
+        m->need_builtin_fallbacks = true;
```

Comment:
What about reverting this to `have_fallbacks`? Then the default initialization will make this false, and you won't need this assignment. I think it's also more natural.